packages/mdc-select/README.md

```diff
@@ -64,17 +59,56 @@ npm install @material/select
 </div>
 ```
 
+The enhanced select uses an MDCMenu component to contain the list of options, but uses the `data-value` attribute
+instead of `value`. 
+
+```html
+<div class="mdc-select">
+  <i class="mdc-select__dropdown-icon"></i>
+  <div class="mdc-select__selected-text" role="combobox"></div>
+  <div class="mdc-select__menu mdc-menu mdc-menu-surface" role="listbox">
+    <ul class="mdc-list">
+      <li class="mdc-list-item mdc-list-item--selected" data-value="" aria-selected="true"></option>
+      <li class="mdc-list" data-value="grains">
+        Bread, Cereal, Rice, and Pasta
+      </li>
+      <li class="mdc-list" data-value="vegetables">
+        Vegetables
+      </li>
+      <li class="mdc-list" data-value="fruit">
+        Fruit
+      </li>
+    </ul> 
+  </div>
+  <label class="mdc-floating-label">Pick a Food Group</label>
+  <div class="mdc-line-ripple"></div>
+</div>
+```
+
+>NOTE: The `data-value` attribute is required for the enhanced select.
+
 ### Styles
 
+For the native select version you can simply include the select Sass file. 
+
 ```scss
 @import "@material/select/mdc-select";
 ```
 
+When using the enhanced select, you will also require the menu components. 
+
+```scss
+@import "@material/list/mdc-list";
+@import "@material/menu-surface/mdc-menu-surface";
+@import "@material/menu/mdc-menu";
+@import "@material/select/mdc-select";
+```
+
```

Comment:
If we're discussing enhanced select here, should we also mention setting width of the select and menu?